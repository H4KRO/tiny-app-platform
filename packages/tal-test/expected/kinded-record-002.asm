main()
  entry:
    FunctionRef       name: func_0
func_0()
  entry:
    Local             name: View
    Local             name: Text
    Literal "text"
    Literal "Hello, World ! 2"
    Literal 1
    MakeRecord
    Literal 2
    MakeArray
    Literal 0
    MakeRecord
    Kinded
