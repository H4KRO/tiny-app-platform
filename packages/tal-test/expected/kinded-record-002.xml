<program>
  <kindedrecord>
    <local />
    <local />
    <record>
      <recordentry>
        <literal />
      </recordentry>
    </record>
  </kindedrecord>
</program>
